.drone/drone.jsonnet

```diff
@@ -20,6 +20,25 @@ local createTag(tag) = {
 	}]
 };
 
+local buildNative(package_name, tag) = {
+  name: "build-native-" + tag,
+	kind: "pipeline",
+	type: "docker",
+	trigger: {branch: [tag]}
+	depends_on: ["create-tag-" + tag],
+	steps: [
+	  {
+		  name: "build-native-debian-package",
+			image: "ubuntu:20.04",
+			environment: {release_type: tag, package_name: package_name},
+			commands: [
+			  "sudo -E apt-get install -y git gnupg pbuilder ubuntu-dev-tools apt-file python3 python3-pip debhelper asciidoctor jq",
+				".drone/scripts/build-native.sh"
+			]
+		}
+	]
+}
+
```

Comment:
Just a placement thing, but could this be moved to be under the `buildAndPublish()` function to match when it's actually executed in the CI pipeline?